src/scripts/setup_test_data/metadata.json

```diff
@@ -1,30 +1 @@
-{
- "1Dbywpp4w2zhXjZPgItrKvlh-raw.zip" : {
-   "recordid": "1Dbywpp4w2zhXjZPgItrKvlh",
-   "substudymemberships": "|mtb-alpha=NNJ18004|",
-   "healthcode": "SdVJHmCC8Dcpd_X-sjEK-wH3",
-   "createdon": "1623421775829",
-   "taskidentifier": "FNAME Test Form 1"
- },
- "ugJWvqPLhnFV-ifu246CeZjZ-raw.zip": {
-   "recordid": "ugJWvqPLhnFV-ifu246CeZjZ",
-   "substudymemberships": "|mtb-alpha=MCH20002|",
-   "healthcode": "vsq7vt_iuT6xjntX09QoQtwk",
-   "createdon": "1624574992880",
-   "taskidentifier": "Vocabulary Form 1"
- },
- "R7mBYI63NpCo3sJa9l0qnpza-raw.zip": {
-   "recordid": "R7mBYI63NpCo3sJa9l0qnpza",
-   "substudymemberships": "|mtb-alpha=NBS22002|",
-   "healthcode": "nzaLFo5nhlqHc1TyUKhvsBP7",
-   "createdon": "1625238358060",
-   "taskidentifier": "MTB Spelling Form 1"
- },
- "Le9-WBxplHhGXo_FLNiY3SaI-raw.zip": {
-   "recordid": "Le9-WBxplHhGXo_FLNiY3SaI",
-   "substudymemberships": "|mtb-alpha=SBM50004|",
-   "healthcode": "XrvTSTKobPJAbxGwZ4sceNBf",
```

Comment:
I see that the metadata.json is pretty different now.  Do we expect this file to change over time?  How does this file affect the code?

Reply:
This file is used by `setup_test_data`, which adds every metadata field to the metadata file and only the partition fields to the rest of the data. @tthyer If we remove recordId from the partition fields this will change so that recordId is still added to every file, it's just not used as a partition field. We may want to consider adding other useful fields to every file. For example, science team is used to having participant identifiers and data groups included with every record. The way the system is designed now requires joins on the metadata to include this information. Something to discuss with science...

Reply:
>Do we expect this file to change over time?

Not really, but it's changing now because Bridge is slowly incorporating all the metadata they planned to include with exporter 3.0 data.